Assets/Scripts/Interaction/PlaceableSurface.cs

```diff
@@ -1,12 +1,13 @@
 ﻿using System;
 using Interaction.Core;
 using Inventory.Custom;
+using Mirror;
 using UnityEngine;
 
 namespace Interaction
 {
     [RequireComponent(typeof(InteractionReceiver))]
-    public class PlaceableSurface : MonoBehaviour, ISingularInteraction
+    public class PlaceableSurface : NetworkBehaviour, ISingularInteraction
```

Comment:
Does this need to be a NetworkBehaviour if it doesn't do any networking?

Reply:
May be leftover from a different attempt to fix it. Can't tell, on phone.

Reply:
Fixed.